scripts/accumulator-server.py

```diff
@@ -33,7 +33,7 @@
 # * Curl users: use -H "Expect:" (default "Expect: 100-continue" used by curl has been problematic
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/5901019aa33640f0984d74aa8d287f3a06c28b05